src/serialization.jl

```diff
@@ -50,7 +50,7 @@ function serialize(geomdata::GeometryData)
 end
 
 intrinsic_transform(g) = IdentityTransformation()
-intrinsic_transform(g::Nullable) = isnull(g) ? IdentityTransformation() : intrinsic_transform(get(g))
+intrinsic_transform(g::Nothing) = isnull(g) ? IdentityTransformation() : intrinsic_transform(get(g))
```

Comment:
I think this method can just be deleted. The fallback on line 52 will just be called for `Nothing`, and the `!isnull` case just forwards to `get(g)`.